### ISSUE TITLE
What I can use in model zoo in object detection with boxes is SSD mobile net v2? 

### ISSUE THREAD

**Issue by kotran88** [tensorflow/models#10156]
Hello 
training process with ssd_mobilenet_v2_fpnlite_640x640 and 320x320 . 
but it has less accuracy so I m trying to implement other models. 

so I got ssd_resnet50_v1_fpn_640x640_coco17_tpu-8 
and changed batch size to 4 num_classes to my label length , 
changed fine_tune_checkpoint
changed fine_tune_checkpoint_type to detection and labels 

with below code. 
it work perfectly with ssd_mobilenet_v2_fpnlite_640x640 and 320x320
but with  ssd_resnet50_v1_fpn_640x640_coco17_tpu-8  and other efficent d1 and fast rcnn model it cause
`
A checkpoint was restored (e.g. tf.train.Checkpoint.restore or tf.keras.Model.load_weights) but not all checkpointed values were used. See above for specific issues. Use expect_partial() on the load status object, e.g. tf.train.Checkpoint.restore(...).expect_partial(), to silence these warnings, or use assert_consumed() to make the check explicit. See https://www.tensorflow.org/guide/checkpoint#loading_mechanics for details.`

what may be the problem? I did just exactly when I try train with mobilenet. 
```

pipeline_config.model.ssd.num_classes = len(labels)
pipeline_config.train_config.batch_size = 4
pipeline_config.train_config.fine_tune_checkpoint = os.path.join(paths['PRETRAINED_MODEL_PATH'], PRETRAINED_MODEL_NAME, 'checkpoint', 'ckpt-0')
pipeline_config.train_config.fine_tune_checkpoint_type = "detection"
pipeline_config.train_input_reader.label_map_path= files['LABELMAP']
pipeline_config.train_input_reader.tf_record_input_reader.input_path[:] = [os.path.join(paths['ANNOTATION_PATH'], 'train.record')]
pipeline_config.eval_input_reader[0].label_map_path = files['LABELMAP']
pipeline_config.eval_input_reader[0].tf_record_input_reader.input_path[:] = [os.path.join(paths['ANNOTATION_PATH'], 'test.record')]
config_text = text_format.MessageToString(pipeline_config)                                                                                                                                                                                                        
with tf.io.gfile.GFile(files['PIPELINE_CONFIG'], "wb") as f:                                                                                                                                                                                                                     
    f.write(config_text)   
```

**Comment by kotran88** [comment]
oh It was my mistake ! I didn't changed model name....:( 
It work with other model than mobilenet!
